content/2018-munich/speakers/richard-hartmann.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Richard Hartmann
+---
+
+## Richard Hartmann
+
+Richard is a Prometheus team member, currently focusing on building a new datacenter for SpaceNet, Richard comes from the networking side of things and has been on call 24/7 for over a decade in his last life. He learned the hard way that precise monitoring which supports operations is key to keeping people sane, and allowing them both sleep and a social life.
```

Comment:
@RichiH You have your name in this sentence twice now. Maybe just put a period after "SpaceNet".